pkg/storage/storage.go

```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/openshift/cluster-image-registry-operator/pkg/storage/oss"
```

Comment:
Put this line near all other `github.com/openshift/cluster-image-registry-operator/pkg/storage/` imports.

And please update GetPlatformStorage.

Reply:
add case condition for GetPlatformStorage function.

Reply:
Imports from `github.com/openshift/cluster-image-registry-operator` should be grouped together, the commented line shouldn't be there.

Reply:
done